<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainWindow</class>
 <widget class="QMainWindow" name="CMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>440</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>440</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BVDEncrypt</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/bvdencrypt/bvdfree.ico</normaloff>:/bvdencrypt/bvdfree.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="CFileSelect" name="fsInputFile" native="true">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>60</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 0, 0);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="CFileSelect" name="fsRulesFile" native="true">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>60</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 170, 0);</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="containerWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Pass phrase</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lePassPhrase"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>User name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leUserName"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Email address</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leEmail"/>
       </item>
       <item>
        <widget class="CFileSelect" name="fsOutputFile" native="true">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>60</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 0);</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnProcess">
           <property name="text">
            <string>Process file</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>1</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="acnQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="acnAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="acnQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="acnAbout">
   <property name="text">
    <string>&amp;About...</string>
   </property>
   <property name="iconText">
    <string>About...</string>
   </property>
   <property name="toolTip">
    <string>About...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CFileSelect</class>
   <extends>QWidget</extends>
   <header>qt_widgets/cfileselect.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>fsInputFile</tabstop>
  <tabstop>fsRulesFile</tabstop>
  <tabstop>lePassPhrase</tabstop>
  <tabstop>leUserName</tabstop>
  <tabstop>leEmail</tabstop>
  <tabstop>fsOutputFile</tabstop>
  <tabstop>btnProcess</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
